UEFITool/uefitool.cpp

```diff
@@ -747,6 +752,23 @@ void UEFITool::toggleBootGuardMarking(bool enabled)
     markingEnabled = enabled;
 }
 
+/* emit double click signal of QListWidget on enter/return key pressed */
+bool UEFITool::eventFilter(QObject* obj, QEvent* event)
+{
+    if (event->type() == QEvent::KeyPress) {
+        QKeyEvent* key = static_cast<QKeyEvent*>(event);
+
+        if (key->key() == Qt::Key_Enter || key->key() == Qt::Key_Return) {
+            QListWidget* list = static_cast<QListWidget*>(obj);
```

Comment:
While I agree that we only bind eventFilter onto QListWidgets, I am not positive Qt may not call it for other widgets too (e.g. children). Even if I am wrong, I believe it is safer to use `qobject_cast` here and check the resulting `list` against `NULL`.

Reply:
done